### DISCUSSION TITLE
ESLint rules as offline docset for Dash

### DISCUSSION THREAD

**Discussion by stefankolb** [eslint/eslint#14913] [General] [answered]
I use [Dash], an API documentation browser, to read documentation of various projects while being offline. I could not find a docset for ESLint, so I was wondering if one exists. If that is not the case, I could give it a shot trying to create a docset for the [ESLint rules], if that is ok with the project maintainer(s). One thing, though: I never created a docset for Dash, so not sure if I would succeed in creating one.

Long story short: Is it okay if I attempt to create a Dash docset for the ESLint rules?

[//]: # "References"

[Dash]: https://kapeli.com/dash
[ESLint rules]: https://eslint.org/docs/rules/

**Comment by aladdin-add** [comment]
I'm not using dash but seems good to have if it'd be helpful for users. 😄

**Comment by nzakas** [comment, marked as answer]
The docs are open source so you are free to use them in whatever you’d like.

**Reply by stefankolb** [reply]
Allright, thanks, I'll give it a try then.

**Reply by stefankolb** [reply]
Just wanted to give a little update on this: I succeeded in creating an ESLint docset and it is part of Dash since this morning (2021-10-25). The script to generate the docset can be found in [the generator repository](https://github.com/stefankolb/dash-eslint-gen).

Currently, only the user guide, developer guide and maintainer guide are part of the docset, but I plan to add more stuff later on.
